L1Trigger/Configuration/python/customiseSettings.py

```diff
@@ -1,6 +1,50 @@
 import os.path
 import FWCore.ParameterSet.Config as cms
 
+def L1TSettingsToCaloStage2Params_2017_v1_9_inconsistent(process):
```

Comment:
This is only used by the experts not in any workflow. 
But good suggestion. thanks.  will do this in the future..will certainly make the documentation shorter.